web_fullscreen/README.rst

```diff
@@ -0,0 +1,34 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==========
+Fullscreen
+==========
+
+Adds a fullscreen mode button that toggles the primary
+and secondary menu.
+
```

Comment:
Somewhere in the readme it should be mentioned that this module is useless if `web_responsive` is installed.

And nice to have: screenshots in the readme, giving a visual effect of the functionality.